DataFormats/SiStripDetId/interface/SiStripDetId.h

```diff
@@ -112,18 +112,18 @@ class SiStripDetId : public DetId {
 // ---------- inline methods ----------
```

Comment:
If implementation of a method is inside header file it is assumed to be inlined even without a declaration.